crates/iota-json-rpc/src/governance_api.rs

```diff
@@ -770,6 +798,8 @@ impl IotaRpcModule for GovernanceReadApi {
 
 #[cfg(test)]
 mod tests {
+    use iota_types::iota_system_state::PoolTokenExchangeRate;
```

Comment:
Add it below the "super" line.

Reply:
Yeah I thought it should be below the "super" line but looks like the `cargo fmt` will force this line to put here.